curriculum/challenges/english/20-upcoming-python/learn-how-to-work-with-numbers-and-strings-by-implementing-the-luhn-algorithm/final.md

```diff
@@ -0,0 +1,68 @@
+---
+id: 6565a536ba1f9f25bd30e88c
+title: Step 100
+challengeType: 20
+dashedName: step-100
+---
+
+# --description--
+
+step 100 instructions
+
+# --hints--
+
+Test 0
+
+```js
+
+```
+
+# --seed--
+
+## --seed-contents--
+
+```python
+--fcc-editable-region--
+def verify_card_number(card_number):
+    # Numbers are mutable
```

Comment:
```suggestion
    # Numbers are immutable
```

Reply:
Haha 😅  Thank you. Honestly, I am pretty sure I know some of this stuff 🙃 

None of this will be in the final project, but I keep it in this `final.md` file for reference as I develop.